curvefs/src/tools/curvefs_tool.h

```diff
@@ -265,7 +272,6 @@ class CurvefsToolRpc : public CurvefsTool {
 
     virtual void SetController() {
         controller_->set_timeout_ms(FLAGS_rpcTimeoutMs);
-        controller_->set_max_retry(FLAGS_rpcRetryTimes);
```

Comment:
You can keep it, and just set max retry times to 0. The framework will retry 3 times by default, but, I think it's also unnecessary.

Reply:
> You can keep it, and just set max retry times to 0. The framework will retry 3 times by default, but, I think it's also unnecessary.

fix